app/helpers/assessment_autograde_core.rb

```diff
@@ -357,7 +357,7 @@ def autogradeDone(submissions, feedback)
     ass_dir = @assessment.folder_path
```

Comment:
nit: not introduced by pr but seems like this variable is unused?

Reply:
Removed